### PR TITLE
Feature/3239/lambda events

### PR DESCRIPTION
**Pull request by garyluu**
For dockstore/dockstore#3239 

First pass at displaying logs.  Still plenty of things wrong with it (major ones mentioned in the TODO statements) which will hopefully be done with https://ucsc-cgl.atlassian.net/browse/SEAB-1367

Using the CircleCI Jar from https://github.com/dockstore/dockstore/pull/3535 .  Must use a "real" Dockstore webservice release before we make a dockstore-ui2 release.

Might want to use a real one before merging this PR too.  🤷 

https://dashboard.cypress.io/projects/1ya4vj/runs/13877/specs myworkflows.ts 0:20-0:23 if you're a machine, u can look at it.  Uploaded a video to Slack on Monday.

### CODE REVIEW

**Comment by denis-yuen** [comment]
> Might want to use a real one before merging this PR too. shrug

Yup

> Going to upload a video on Monday to slack.

:+1:

File: src/app/myworkflows/sidebar-accordion/github-apps-logs/github-apps-logs.component.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
+<h1 mat-dialog-title>GitHub App Logs</h1>
+<div mat-dialog-content>
+  <app-loading [loading]="loading">
+    <span *ngIf="showContent === 'error'">There were problems retrieving GitHub App logs for this organization.</span>
```
[the remark this answers is not in the document]
Comment (garyluu) [reply src/app/myworkflows/sidebar-accordion/github-apps-logs/github-apps-logs.component.html:4]: it's of type:
`showContent: 'table' | 'error' | 'empty' | null;`

If your IDE supports it, you should see issues if using none of those values.

File: src/app/myworkflows/sidebar-accordion/github-apps-logs/github-apps-logs.component.ts [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+      if (lambdaEvents.length === 0) {
+        this.showContent = 'empty';
+      } else {
+        this.showContent = 'table';
+      }
+    }
+  }
+
+  applyFilter(event: Event) {
+    const filterValue = (event.target as HTMLInputElement).value;
```
Comment (coverbeck) [inline src/app/myworkflows/sidebar-accordion/github-apps-logs/github-apps-logs.component.ts:83]: Is there not a more Angular-ly way of doing this? Or is that covered by your TODO on filtering by dates.

Comment (garyluu) [reply src/app/myworkflows/sidebar-accordion/github-apps-logs/github-apps-logs.component.ts:83]: Seems fairly Angular to me I think https://angular.io/guide/user-input#get-user-input-from-the-event-object

Or did you have ngModel in mind?

Comment (coverbeck) [reply src/app/myworkflows/sidebar-accordion/github-apps-logs/github-apps-logs.component.ts:83]: Given your link, it seems OK. You know Angular better than I, but it felt odd to me off-hand to be casting to a DOM element.

**Pull request merged**